<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="19" hostname="DESKTOP-DJ4FDME" failures="4" tests="5" name="Test(failed)" time="0.175" errors="0" timestamp="2024-10-07T22:20:41 IST">
  <testcase classname="demoTestNG.FirstScenario9" name="third_testcase" time="0.034">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at demoTestNG.FirstScenario9.third_testcase(FirstScenario9.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- third_testcase -->
  <testcase classname="demoTestNG.FirstScenario9" name="forth_testcase" time="0.0">
    <skipped/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="demoTestNG.FirstScenario6" name="third_testcase" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at demoTestNG.FirstScenario6.third_testcase(FirstScenario6.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- third_testcase -->
  <testcase classname="demoTestNG.FirstScenario" name="first_testcase" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at demoTestNG.FirstScenario.first_testcase(FirstScenario.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.FirstScenario8" name="sixth_testcase" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at demoTestNG.FirstScenario8.sixth_testcase(FirstScenario8.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="demoTestNG.FirstScenario6" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="demoTestNG.FirstScenario9" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="demoTestNG.FirstScenario8" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="demoTestNG.FirstScenario6" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.FirstScenario" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="demoTestNG.FirstScenario" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="demoTestNG.FirstScenario6" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="demoTestNG.FirstScenario8" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.FirstScenario9" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.FirstScenario9" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="demoTestNG.FirstScenario6" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="demoTestNG.FirstScenario6" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="demoTestNG.FirstScenario" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="demoTestNG.FirstScenario9" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="demoTestNG.FirstScenario8" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="demoTestNG.FirstScenario" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="demoTestNG.FirstScenario" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="demoTestNG.FirstScenario8" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="demoTestNG.FirstScenario8" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
</testsuite> <!-- Test(failed) -->
